users/tests/test_oauth_login.py

```diff
@@ -0,0 +1,251 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+from rest_framework_simplejwt.tokens import RefreshToken
+import requests_mock
+from ..models import UserRole
```

Comment:
Use an absolute import (from users.models import UserRole) to avoid import issues if the test structure changes.